forge-gui/res/cardsfolder/upcoming/braids_arisen_nightmare.txt

```diff
@@ -0,0 +1,13 @@
+Name:Braids, Arisen Nightmare
+ManaCost:1 B B
+Types:Legendary Creature Nightmare
+PT:3/3
+T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigSac | TriggerDescription$ At the beginning of your end step, you may sacrifice an artifact, creature, enchantment, land, or planeswalker. If you do, each opponent may sacrifice a permanent that shares a card type with it. For each opponent who doesn't, that player loses 2 life and you draw a card.
+SVar:TrigSac:DB$ Sacrifice | Optional$ True | SacValid$ Artifact,Creature,Enchantment,Land,Planeswalker | Amount$ 1 | RememberSacrificed$ True | SubAbility$ DBRepeatEach
+SVar:DBRepeatEach:DB$ RepeatEach | RepeatSubAbility$ DBMaySac | RepeatPlayers$ Player.Opponent | SubAbility$ DBCleanup
+SVar:DBMaySac:DB$ Sacrifice | Defined$ Player.IsRemembered | SacValid$ Permanent.sharesCardTypeWith Remembered | RememberSacrificed$ True | Optional$ True | SubAbility$ DBLoseLife
```

Comment:
maybe `Permanent.sharesCardTypeWith RememberedCard` - always a bit nerve-racking with different objects remembered

Reply:
Made the change, thank you!